spec/index.html

```diff
@@ -350,9 +350,9 @@ <h3 id="message-authorization"><a class="toc-anchor" href="#message-authorizatio
-<li>If attestation of an object is permitted the <code>payload</code> <strong><strong>MAY</strong></strong> include an <code>attestationCid</code> property, and its value <strong><strong>MUST</strong></strong> be the stringified <a path-0="docs.ipfs.io"path-1="concepts"path-2="content-addressing"path-3="#identifier-formats"href="https://docs.ipfs.io/concepts/content-addressing/#identifier-formats" >Version 1 CID</a> of the <a path-0="github.com"path-1="ipld"path-2="specs"path-3="blob"path-4="master"path-5="block-layer"path-6="codecs"path-7="dag-cbor.md"href="https://github.com/ipld/specs/blob/master/block-layer/codecs/dag-cbor.md" >DAG CBOR</a> encoded <code>attestation</code> string.</li>
+<li>The object <strong><strong>MUST</strong></strong> include a <code>descriptorCid</code> property, and its value <strong><strong>MUST</strong></strong> be the stringified <a path-0="docs.ipfs.io"path-1="concepts"path-2="content-addressing"path-3="#identifier-formats"href="https://docs.ipfs.io/concepts/content-addressing/#identifier-formats" >Version 1 CID</a> of the <a path-0="github.com"path-1="ipld"path-2="specs"path-3="blob"path-4="master"path-5="block-layer"path-6="codecs"path-7="dag-cbor.md"href="https://github.com/ipld/specs/blob/master/block-layer/codecs/dag-cbor.md" >DAG CBOR</a> encoded <code>descriptor</code> object.</li>
+<li>The object <strong><strong>MAY</strong></strong> include a <code>permissionsGrantCid</code> property, and its value <strong><strong>MUST</strong></strong> be the stringified <a path-0="docs.ipfs.io"path-1="concepts"path-2="content-addressing"path-3="#identifier-formats"href="https://docs.ipfs.io/concepts/content-addressing/#identifier-formats" >Version 1 CID</a> of the <a path-0="github.com"path-1="ipld"path-2="specs"path-3="blob"path-4="master"path-5="block-layer"path-6="codecs"path-7="dag-cbor.md"href="https://github.com/ipld/specs/blob/master/block-layer/codecs/dag-cbor.md" >DAG CBOR</a> encoded  <a href="#grant" >Permission Grant</a> being invoked.</li>
+<li>If attestation of an object is permitted the <code>payload</code> <strong><strong>MAY</strong></strong> include a <code>attestationCid</code> property, and its value <strong><strong>MUST</strong></strong> be the stringified <a path-0="docs.ipfs.io"path-1="concepts"path-2="content-addressing"path-3="#identifier-formats"href="https://docs.ipfs.io/concepts/content-addressing/#identifier-formats" >Version 1 CID</a> of the <a path-0="github.com"path-1="ipld"path-2="specs"path-3="blob"path-4="master"path-5="block-layer"path-6="codecs"path-7="dag-cbor.md"href="https://github.com/ipld/specs/blob/master/block-layer/codecs/dag-cbor.md" >DAG CBOR</a> encoded <code>attestation</code> string.</li>
```

Comment:
`an attestationCid` was correct...
```suggestion
<li>If attestation of an object is permitted the <code>payload</code> <strong><strong>MAY</strong></strong> include an <code>attestationCid</code> property, and its value <strong><strong>MUST</strong></strong> be the stringified <a path-0="docs.ipfs.io"path-1="concepts"path-2="content-addressing"path-3="#identifier-formats"href="https://docs.ipfs.io/concepts/content-addressing/#identifier-formats" >Version 1 CID</a> of the <a path-0="github.com"path-1="ipld"path-2="specs"path-3="blob"path-4="master"path-5="block-layer"path-6="codecs"path-7="dag-cbor.md"href="https://github.com/ipld/specs/blob/master/block-layer/codecs/dag-cbor.md" >DAG CBOR</a> encoded <code>attestation</code> string.</li>
```